code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -617,14 +617,17 @@ emp_act
 	visible_message("<span class='danger'>[I] embeds itself in [src]'s [L.name]!</span>","<span class='userdanger'>[I] embeds itself in your [L.name]!</span>")
 	return TRUE
 
-/mob/living/carbon/human/proc/make_bloody_hands(list/blood_dna, b_color)
+/mob/living/carbon/human/proc/make_bloody_hands(list/blood_dna, b_color, amount=null)
```

Comment:
No need to specify, it defaults to `null`. Also, can you document this proc's arguments?
```suggestion
/mob/living/carbon/human/proc/make_bloody_hands(list/blood_dna, b_color, amount)
```